spec/rubocop/cop/rspec/redundant_predicate_matcher_spec.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::RedundantPredicateMatcher do
+  it 'registers an offense when using `be_all`' do
+    expect_offense(<<~RUBY)
+      expect(foo).to be_all be_odd
+                     ^^^^^^^^^^^^^ Use `all` instead of `be_all`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      expect(foo).to all be_odd
+    RUBY
+  end
```

Comment:
Yet another
```ruby
        expect(deployment_refs_exist?).to be_all(false)
                                          ^^^^^^^^^^^^^
```

Reply:
Ditto. I change to not offense for this case.